ipv-core/src/main/java/com/ibm/whc/deid/providers/masking/BinningMaskingProvider.java

```diff
@@ -74,4 +87,43 @@ public String mask(String identifier) {
 
     return String.format(this.format, Long.valueOf(lowerBase), Long.valueOf(higherBase));
   }
+  
+  public String maskConstant(String identifier) {
+    if (identifier == null) {
+      debugFaultyInput("identifier");
+      return null;
+    }
+    
+    if (this.useSingleBucketOverThreshold==false || this.useSingleBucketUnderThreshold==false) {
+    	debugFaultyInput("identifier");
+        return null;
+    } 
+    
+    String range="";
+    double value;
+    double upperRange=this.singleBucketOverThresholdValue;
+    double lowerRange=this.singleBucketUnderThresholdValue;
+    
+    try {
+      value = Double.parseDouble(identifier);
+    } catch (NumberFormatException e) {
+      // For this provider, we do not return a random value
+      return applyUnexpectedValueHandling(identifier, null);
+    }
+    
+    if(value>=upperRange) {
+    	range=String.valueOf((int)upperRange);
+    	range=range+"+";
+    }
+     
+    else if(value<lowerRange) {
```

Comment:
Same for this range - if the value is less than the lower threshold and the boolean for lower is true, return the given lower replacement value.